Samples/MediaPlayerCPP/MediaPlayerCPP.vcxproj

```diff
@@ -8,6 +8,7 @@
     <AppContainerApplication>true</AppContainerApplication>
     <ApplicationType>Windows Store</ApplicationType>
     <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion Condition="'$(VisualStudioVersion)' == '16.0'">10.0</WindowsTargetPlatformVersion>
```

Comment:
Maybe use `Condition="'$(VisualStudioVersion)' &gt;= '16.0'"`, so this will work in VSNext as well.